### PR TITLE
Update Dutch translations

### PR DESCRIPTION
**Pull request by djdirty60**
Updated

### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](../../master/Contributing.md) document.
* [x] I made sure there wasn't another [Pull Request](../../../pulls) opened for the same update/change
* [x] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [x] Feature change (non-breaking change which changes behaviour of an existing functionality)
- [ ] Improvement (non-breaking change which improves functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactor (non-breaking performance or readability improvements)

### CODE REVIEW

**Comment by djdirty60** [comment]
Updated

File: resources/language/resource.language.nl_nl/strings.po [hunk 1]
```diff
@@ -1030,80 +1030,80 @@ msgstr "Op genre/subgenre ID"
...
 msgctxt "#30610"
 msgid "Use system-based encryption for credentials"
-msgstr ""
+msgstr "Gebruik systeemgebaseerde versleuteling voor referenties"
 
 # Help description for setting id 30610
 msgctxt "#30611"
 msgid "[B]If disabled it will cause a security leak[/B]. If your operating system does not support it, you will be prompted to login each time you reboot the device, [B]only[/B] in this case disable it. If you change this setting you will be prompted to login again in the next time you reboot."
-msgstr ""
+msgstr "[B]Indien uitgeschakeld, veroorzaakt dit een beveiligingslek[/B].  Als uw besturingssysteem dit niet ondersteunt, wordt u elke keer dat u het apparaat opnieuw opstart gevraagd om in te loggen. Schakel het in dit geval alleen [B][/B] uit.  Als u deze instelling wijzigt, wordt u de volgende keer dat u opnieuw opstart, gevraagd opnieuw in te loggen."
```
Comment (CastagnaIT) [inline resources/language/resource.language.nl_nl/strings.po:1087]: something wrong with `[B][/B]` contains no words

**Review by CastagnaIT — CHANGES_REQUESTED** [review]

**Comment by CastagnaIT** [comment]
`Translations / Check translations (matrix) (pull_request)` is failed

resources/language/resource.language.nl_nl/strings.po:156: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:160: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:164: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:168: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:172: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:176: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:180: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:184: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:262: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:330: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:614: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:618: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:622: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:638: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:1046: this message is untranslated
resources/language/resource.language.nl_nl/strings.po:1062: this message is untranslated

**Comment by CastagnaIT** [comment]
if request changes will be not addressed in few days this PR will be closed

**Pull request closed without merging**